client/reader/recent/style.scss

```diff
@@ -13,17 +19,20 @@
 
 	%column-shared {
 		background: var( --color-surface );
-		border-radius: 8px; /* stylelint-disable-line scales/radii */
+		border-radius: $feed-card-border-radius;
 		box-shadow: 0 0 17.4px 0 rgba( 0, 0, 0, 0.05 );
-		height: 100%;
-		overflow-y: auto;
-		overflow-x: hidden;
```

Comment:
Removing these styles from common styling so that we can implement separate scroll behaviors for both sections.